tools/sdk-ai-bots/AzureSdkQaBot/LocalPkg/NuGet_Package_PlaceHolder

```diff
@@ -0,0 +1 @@
+Put the Microsoft.TeamsAI.<version>.nupkg here to be referenced by sample project.
```

Comment:
Do we need to merge those nuget packages above into the repo?

Reply:
I removed the unused packages. For the only one remained, we need to check in it as it might hit issues to fetch it from github feed. I will update the code to use the GA library later on when we can get rid of this package from the repo.

Reply:
Is the package not available on a feed?